llvm/test/Transforms/LoopVectorize/AArch64/sve-widen-phi.ll

```diff
@@ -27,43 +27,43 @@ define void @widen_ptr_phi_unrolled(ptr noalias nocapture %a, ptr noalias nocapt
 ; CHECK-NEXT:    [[N_VEC:%.*]] = and i64 [[DOTNEG]], [[N]]
 ; CHECK-NEXT:    [[TMP3:%.*]] = shl i64 [[N_VEC]], 3
 ; CHECK-NEXT:    [[IND_END:%.*]] = getelementptr i8, ptr [[C:%.*]], i64 [[TMP3]]
-; CHECK-NEXT:    [[TMP25:%.*]] = call i64 @llvm.vscale.i64()
-; CHECK-NEXT:    [[TMP26:%.*]] = shl nuw nsw i64 [[TMP25]], 3
+; CHECK-NEXT:    [[TMP4:%.*]] = call i64 @llvm.vscale.i64()
```

Comment:
At a glance this whole file looks like just renumbering; if the old test still passes, don't touch it?

Reply:
So, what happened is that this test failed when I made an earlier edit, so I tried to update the tests. But then when I realized a bug happened, I fixed the bug and regenerated the test to revert back. I can undo this change.